build.sbt

```diff
@@ -649,11 +649,13 @@ lazy val root = (project in file("."))
     publish := {},
     publishLocal := {},
     commands ++= ScriptCommands.all,
-    // Generate (Product|TupleN|Function|AbstractFunction)*.scala files. Once the ANT build is gone,
-    // we should move them into a managedSources dir instead of overwriting sources checked into git.
-    commands += Command.command("genprod") { state =>
-      val dir = (((baseDirectory in ThisBuild).value) / "src" / "library" / "scala").getPath
-      genprod.main(Array(dir))
+    // Generate (Product|TupleN|Function|AbstractFunction)*.scala files and scaladoc stubs for all
+    // AnyVal sources. Once the ANT build is gone, we should move them into a managedSources dir instead
+    // of overwriting sources checked into git.
```

Comment:
Ah, that's unfortunate. We could easily include the generated sources in source JARs but if the links go directly to github, we're out of luck. Is there a way to disable source links selectively? It's only a stub that doesn't really add any value on top of the scaladoc page, so we could just remove the link if that's possible.